### ISSUE TITLE
Parse error when using Postgres JSON syntax

### ISSUE THREAD

**Issue by tmtron** [sql-formatter-org/sql-formatter#291]
**Describe the bug**
A `Parsing Error` occurs for the following query (language `PostgreSQL`):
```sql
SELECT id, name FROM tabe WHERE jsoncol @? 'strict $.** ? (@.id == 2 )'
```

**Expected behavior**
Should format the query

**Actual behavior**
```
Parse error: Unexpected "? 'strict $.** ? (@.id == 2 )'"

Stack Trace:

Error: Parse error: Unexpected "? 'strict $.** ? (@.id == 2 )'"
    at E.value (https://unpkg.com/sql-formatter@latest/dist/sql-formatter.min.js:1:37270)
    at N.value (https://unpkg.com/sql-formatter@latest/dist/sql-formatter.min.js:1:29851)
    at Module.OS (https://unpkg.com/sql-formatter@latest/dist/sql-formatter.min.js:1:208492)
    at HTMLTextAreaElement.format (https://sql-formatter-org.github.io/sql-formatter/index.js:5
```

**Usage**

- How are you calling / using the library?
  - this can be reproduced in the [online-formatter](https://sql-formatter-org.github.io/sql-formatter/)
- What SQL language(s) does this apply to?
  - set the for language to `PostgreSQL`
- Which SQL Formatter version are you using?
  -  8.0.2

**Comment by inferrinizzard** [comment]
Looks like a broken interaction with the @? operator, will dig into this

**Comment by nene** [comment]
Looks like `@?` is another one of the myriad of Postgres operators that we have missed: https://www.postgresql.org/docs/current/functions-json.html

Really should go through all the operators docs and check them all, instead of fixing one-by-one as the bug reports come in :)

**Comment by nene** [comment]
Fix released in 8.1.0

**Comment by tmtron** [comment]
confirmed that it works in version 8.1.0 - thanks!
